### ISSUE TITLE
Add GPG and SSH key to GitHub user data source

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-github#15]
_This issue was originally opened by @mootpt as hashicorp/terraform#15211. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

Add GPG and SSH key to GitHub user data source:

```
data "github_user" "mootpt" {
  username = "mootpt"
}

output "mootpt-gpg" {
  value = "${data.github_user.mootpt.gpg_key}"
}

output "mootpt-ssh" {
  value = "${data.github_user.mootpt.ssh_key}"
}
```

**Comment by grv87** [comment]
Duplicate of #4

**Comment by stack72** [comment]
Closed via #2
